### ISSUE TITLE
UNINSTALL leaves behind misc. cluster-scoped resources

### ISSUE THREAD

**Issue by dgrove-oss** [project-codeflare/mlbatch#21]
Should also give commands for removing priorities, cluster roles, etc. so that there are no mlbatch entities left.

**Comment by tardieu** [comment]
Agreed. The current instructions focus on removing all running pods and only the resources responsible for or preventing the removal of these running pods.
